src/components/Dropdown/Dropdown.tsx

```diff
@@ -26,6 +26,8 @@ export interface DropdownProps {
   label?: string;
   alignment?: DROPDOWN_ALIGNMENT;
   placement?: Placement;
+  ariaLabel?: string;
+  ariaLabelledBy?: string;
```

Comment:
We're actually trying to start using `aria-` attributes the same way we would regularly use them in regular html.
Meaning naming the props: `aria-label` and `aria-labelledby`.
I should have told you this before, sorry 😬 
Other than that looks great, thanks!